theano/gof/tests/test_destroyhandler.py

```diff
@@ -47,6 +47,10 @@ def MyConstant(data):
 
 class MyOp(Op):
 
+    __props__ = ("nin", "name", "vmap", "dmap", "nout",
+                 "destroyhandler_tolerate_same",
+                 "destroyhandler_tolerate_aliased")
+
```

Comment:
I have committed this a couple of months back, but haven't pushed it so I didn't notice this in the PR. Do I revert it or keep it ?